docs: add minimal docs for self-managed infrastructure (#2458)
edgelesssys/constellation: docs/docs/getting-started/first-steps.md

-->

3. Create the cluster. `constellation create` uses options set in `constellation-conf.yaml`.

>     If you want to manually manage your cloud resources, for example by using [Terraform](../reference/terraform.md), follow the corresponding instructions in the [Create workflow](../workflows/create.md).

Reviewer: Are the contents of terraform.md up to date?

Author: Yes, they are

>     If you want to manually use [Terraform](../reference/terraform.md) for managing the cloud resources instead, or want completely self-managed infrastructure, follow the corresponding instructions in the [Create workflow](../workflows/create.md).

Reviewer 2: ```suggestion
    If you want to manually manage your cloud resources, for example by using [Terraform](../reference/terraform.md), follow the corresponding instructions in the [Create workflow](../workflows/create.md).
```
IMO manually using Terraform implies self-managed infrastructure, this rewording fits a bit better.
